apps/nrm/behaviours/save-and-exit.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 const moment = require('moment');
+const config = require('../../../config');
 
 module.exports = superclass => class extends superclass {
 
   locals(req, res) {
     const superlocals = super.locals(req, res);
     const data = Object.assign({}, {
-      reportExpiration: moment(req.sessionModel.get('created_at')).format('DD MMMM YYYY'),
+      reportExpiration: moment(req.sessionModel.get('created_at'))
+          .add(config.reports.deletionTimeout, 'days').format('DD MMMM YYYY'),
```

Comment:
great spot and great work

so the original code, doesn't it have tests?
I think we need unit tests for this